src/components/Header.js

```diff
@@ -1,7 +1,15 @@
-import React from "react";
-import Search from "./Search";
+import React from "react"
+import Search from "./Search"
+
+function Header({ listings, setSearchedListings }) {
+  const searchListings = (search) => {
+    setSearchedListings(
+      listings.filter((listing) =>
+        listing.description.toLowerCase().includes(search.toLowerCase())
+      )
+    )
+  }
```

Comment:
This works, but if you do it in the parent of the component, as it doesn't need to be done in the child, then it'd be one less prop to pass down, which is nice.